misc/fluentd/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine@sha256:58e1a1bb75db1b5a24a462dd5e2915277ea06438c3f105138f97eb53149673c4
```

Comment:
We can, but in that way we lose the ability to config the fluentd driver, like the log file name, and filter and so on. we can just to follow the default format of the logging of `fluentd/fluentd` and will get rid of the docker build process to make this work. Which one do you prefer?